src/main/resources/swagger/api-docs.yaml

```diff
@@ -2175,6 +2175,20 @@ paths:
           description: Internal server error unlinking NIH account
           content: {}
       x-passthrough: false
+  /api/nih/resources:
+    get:
+      tags:
+        - NIH
+      summary: Retrieves info about a user's NIH Resources
+      operationId: NIHResources
+      responses:
+        200:
+          description: Successfully retrieved NIH Resources (without Linked NIH Username)
```

Comment:
nit: remove "(without Linked NIH Username)"; that only makes sense if you are comparing this API to /api/nih/status